pilot/pkg/model/service.go

```diff
@@ -84,6 +86,9 @@ type Service struct {
 	// or use the passthrough model (i.e. proxy will forward the traffic to the network endpoint requested
 	// by the caller)
 	Resolution Resolution
+
+	// CreationTime records the time this service was created, if available.
+	CreationTime time.Time `json:"creationTime,omitempty"`
```

Comment:
well, not a huge deal - but modified time is what matters, you can have a service created a month ago, that gets modified with something invalid - it should still be ignored.

Reply:
Ultimately this value is coming from k8s.  Will k8s supply the original time the resource was created or the last time it was updated?